addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -1445,6 +1448,8 @@ var SnippetEditor = Widget.extend({
 
         delete this.$dropZones;
         delete this.dragState;
+
+        window.parent.removeEventListener('mouseup', this.onParentWindowMouseup, {once: true});
```

Comment:
I don't think the remove function is supposed to receive the once parameter?

Reply:
I think it's better to put it like the [documentation](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/removeEventListener) says this:
`It's worth noting that some browser releases have been inconsistent on this, and unless you have specific reasons otherwise, it's probably wise to use the same values used for the call to addEventListener() when calling removeEventListener().`
What do you think ? Maybe overkill ...

Reply:
Ok, maybe, although the same page says that `once` is not a valid value, but ok :wink: As long as you tested the event handler is indeed removed, that's fine :+1: